icu4c/source/common/rbbi.cpp

```diff
@@ -1230,7 +1239,7 @@ RuleBasedBreakIterator::getLanguageBreakEngine(UChar32 c) {
 
     int32_t i = fLanguageBreakEngines->size();
     while (--i >= 0) {
-        lbe = (const LanguageBreakEngine *)(fLanguageBreakEngines->elementAt(i));
```

Comment:
this is surely wrong, LanguageBreakEngine should be immutable and not changed once created, you need to keep it as a const pointer.

Reply:
Fixed